tests/Feature/WhitelistTest.php

```diff
@@ -15,6 +15,13 @@ public function testShouldAllow()
         $this->assertEquals('next', (new Whitelist())->handle($this->app->request, $this->getNextClosure()));
     }
 
+    public function testShouldAllowMultiple()
+    {
+        config(['firewall.whitelist' => ['127.0.0.0/24:0.0.0.0']]);
```

Comment:
This would fail because nothing after it actually parses it out into a list of IPs. It needs to be parsed out and turned into an array before being stored in the config value. This is what my pull request does.

Reply:
Also, we should consider that separating these by a `:` will be problematic for ipv6.